meshroom/ui/qml/GraphEditor/Node.qml

```diff
@@ -108,6 +110,7 @@ Item {
         drag.threshold: 2
         hoverEnabled: true
         acceptedButtons: Qt.LeftButton | Qt.RightButton
+        onPositionChanged: root.mousePosition = mapToItem(mouseArea, mouseArea.mouseX, mouseArea.mouseY)
```

Comment:
Probably no need to map mouseArea to itself.

Reply:
In that case, better to keep declarative code (instead of imperative  code).

Reply:
This line is actually not needed anymore since `property point mousePosition: Qt.point(mouseArea.mouseX, mouseArea.mouseY)` does the job by itself.